.changeset/clever-llamas-listen.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": major
+"@khanacademy/perseus-editor": major
+---
+
+Update to use wonder-blocks deps after migrating wonder-blocks to TS
```

Comment:
I just realized that these can be minor bumps and `math-input` can be a patch bump since we aren't changing the API of these packages.  The major bumps will occur as part of the TS migration.